### DISCUSSION TITLE
Missing generated datafetcher examples for Kotlin

### DISCUSSION THREAD

**Discussion by jmbarroso** [Netflix/dgs-framework#134] [Q&A] [answered]
Hi,

First of all, thanks for the DGS framework. I'm super excited to try to use it. 

I wanted to ask if there is any special config to get the datafetchers autogenerated code (Kotlin) in the _generated-examples_ folder. 

I couldn't make it work in my project or the [Kotlin example project](https://github.com/Netflix/dgs-examples-kotlin). 
It did work on the [Java example](https://github.com/Netflix/dgs-examples-java), but not in Kotlin. Is there any specific config needed? 

Thank you!

**Comment by paulbakker** [comment, marked as answer]
This was never implemented for Kotlin. 
More generally, the generated datafetchers never saw a lot of use at Netflix, so we're actually wondering what we should do with this feature. Feedback around this is definitely welcome.

**Reply by jmbarroso** [reply]
Thanks for the quick answer. That makes sense.

I wanted to see how much boilerplate code could be skipped using the generated code. But I'm starting my implementation so maybe I will end in the same place, not using it. :)

**Comment by srinivasankavitha** [comment]
You need to set the language to KOTLIN, like so: 
```language = "kotlin"```
It's not set currently in your project, if you want to get the generated POJOs.

**Reply by paulbakker** [reply]
Except that it won't generate datatfetchers :)

**Reply by jmbarroso** [reply]
Actually, it generated the Kotlin types without that property. It picked up the language from the project config as indicated in the doc. Thanks for the answer.
